File: Module-06/01-colonnes-df/01-lignes.py
from pyspark.sql import Row

uneLigne = Row(1,"Isabelle","BIZOÏ")
uneAutre = Row(2,"Razvan","BIZOÏ")
listeLignes = [uneLigne,uneAutre]
personnes = spark.createDataFrame(listeLignes,["Id","Prenom","Nom"])
personnes.show()

File: Module-06/01-colonnes/01-lignes.scala
import org.apache.spark.sql.Row

val uneLigne = Row(1,"Isabelle","BIZOÏ")
val uneAutre = Row(2,"Razvan","BIZOÏ")

val lignes = Seq((2,"Razvan","BIZOÏ"),(1,"Isabelle","BIZOÏ"))
val personnes = lignes.toDF("Id","Prenom","Nom")
personnes.printSchema()
personnes.show()
